client/src/components/Layout/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useMemo } from "react";
+import { Layout } from "antd";
+import { Outlet } from "react-router-dom";
+import Navbar from "components/Navbar";
+import Footbar from "components/Footbar";
+// import ErrorToast from "components/ErrorToast";
+import { useMediaQuery } from "hooks/useMediaQuery";
+
+const { Header, Content, Footer } = Layout;
+
+export default function MainLayout() {
+    const isMobile = useMediaQuery("(max-width: 375px)");
+
+    const headerStyle = useMemo(
```

Comment:
why set header, content, footer style details inside index.jsx instead of defining them in css file under Layout folder?

Reply:
> why set header, content, footer style details inside index.jsx instead of defining them in css file under Layout folder?

You can do that either way. I just don't want to set too much `className` in the main  layout

Reply:
why use useMemo here? if we move the style details in css file, do we still need it?

Reply:
> why use useMemo here? if we move the style details in css file, do we still need it?

We can move all style data to css file and remove it from the jsx file. But I suggest keeping it until everything's finished.